modules/gui/src/com/haulmont/cuba/gui/components/actions/FilterApplyAction.java

```diff
@@ -21,7 +21,7 @@
 import com.haulmont.cuba.gui.components.ListComponent;

```

Comment:
Sorry, this file got here by an accident - I just marked this with deprecated annotation and used auto-formatting.